dbt/README.md

```diff
@@ -625,12 +669,160 @@ do so, you have two options:
+and a tax year defined by `$TAXYR`:
+
+```
+python3 scripts/export_models.py --select qc.vw_change_in_ahsap_values --where "taxyr = '$TAXYR' and township_name = '$TOWNSHIP_NAME'"
+```
+
+The script will output the reports to the `dbt/export/output/` directory, and will print the
+names of the reports that it exports during execution.
+
+#### Adding QC reports
+
+Since QC reports are built on top of models, adding a new QC report can be as simple
+as adding a new model and exporting it using the [`export_models`
+script](https://github.com/ccao-data/data-architecture/blob/master/dbt/scripts/export_models.py).
+You should default to adding your model to the `qc` schema and subdirectory, unless there is
+a good reason to define it elsewhere. For details on how to add a model, see
+[➕ How to add a new model](#-how-to-add-a-new-model).
+
+There are a number of configuration options that allow you to control the format of your
+model during and after export:
+
+* **Tagging**: [Model tags](https://docs.getdbt.com/reference/resource-configs/tags) are not
+  required for QC reports, but they are helpful in cases where we need to export more than one
+  report at a time. For example, Valuations typically requests all of the town close QC
+  reports at the same time, so we tag each model with the `qc_report_town_close` tag such that
+  we can select them all at once when running the `export_models` script using
+  `--select tag:qc_report_town_close`. For consistency, prefer tags that start with the `qc_*`
+  prefix, but beware not to use the `test_qc_*` prefix, which is instead used for [QC
+  tests](#adding-qc-tests).
+* **Filtering**: Since the `export_models` script can filter your model using the `--where`
+  option, you should define your model such that it selects any fields that you want to use
+  for filtering in the `SELECT` clause. It's common to filter reports by `taxyr` and
+  one of either `township_name` or `township_code`.
+* **Formatting**: You can set a few different optional configs on the `meta` attribute of
+  your model's schema definition in order to control the format of the output workbook:
+    * **`meta.export_name`**: The base name that the script will use for the output file, not
+      including the file extension. The script will output the file to
+      `dbt/export/output/{meta.export_name}.xlsx`.
+    * **`meta.export_template`**: A path to an Excel template to populate with data, relative
+      to the [`dbt/export/templates/`
+      directory](https://github.com/ccao-data/data-architecture/tree/master/dbt/export/templates).
+      The script will read this template from `dbt/export/templates/{meta.export_template}`.
+      In contrast to `meta.export_name`, this option _does_ expect you to include the file
+      extension in the template name, to allow you more flexibility over the format of the
+      template files you use. Templates are useful if you want to apply custom headers,
```

Comment:
I wonder if it would be simpler if we just standardized both configs as requiring or not requiring file extensions? It doesn't feel super urgent to me since I doubt most reports will use both of these options anyway, but I'm curious what you think.

Reply:
What's the context in which we use/will use template formats other than `.xlsx`?

Reply:
Hm, good point -- I can't think of one. I switched things up in d42e97edd1d9e8a65a4bb18e57779173c79a5894 so that `meta.export_template` doesn't expect a file extension, either.